source/MRViewer/MRRenderVolumeObject.h

```diff
@@ -8,7 +8,7 @@
 
 namespace MR
 {
-class RenderVolumeObject : public IRenderObject
+class MRVIEWER_CLASS RenderVolumeObject : public IRenderObject
```

Comment:
do we need export this class?

Reply:
why we don't need this (unlike others object classes)?